### ISSUE TITLE
Conflict Between HtmlSanitizer and Bunit Due to AngleSharp Version

### ISSUE THREAD

**Issue by HarshitDubey-Raik** [mganss/HtmlSanitizer#558]
I am encountering a version conflict while writing unit tests using Xunit in C#. I have a project named Test which references another project named Worker. The Worker project includes HtmlSanitizer.
In the Test project, I need to install Bunit, which requires AngleSharp version >= 1.1.2.

This version mismatch is causing conflicts, and I am unable to resolve this dependency issue.

Following is the error being displayed while installing
```CShap
Version conflict detected for AngleSharp. 
Install/reference AngleSharp 1.1.2 directly to project Tests to resolve this issue. 
Tests -> bunit 1.30.3 -> bunit.web 1.30.3 -> AngleSharp (>= 1.1.2) 
Tests -> Worker -> HtmlSanitizer 8.0.865 -> AngleSharp (= 0.17.1).
```

Any insights or suggestions on resolving this dependency conflict would be greatly appreciated. 
Thank you!

**Comment by mganss** [comment]
Try the latest prerelease version of HtmlSanitizer. It's compatible with AngleSharp 1.1.2. See also #491

**Comment by HarshitDubey-Raik** [comment]
@mganss Actually the application using the HtmlSanitizer is in production, so cannot use this pre-release beta version, if you can please confirm till when can we except the proper release for the same ?

**Comment by tiesont** [comment]
@HarshitDubey-Raik AngleSharp.Css is the hold-up, last I was aware - waiting on a 1.0.0 release, so that there can be a fair amount of confidence that the API won't change significantly (which has been a bit of a problem with AngleSharp in the past, as noted in the linked issue).

**Comment by tiesont** [comment]
Worth noting? https://github.com/AngleSharp/AngleSharp.Css/issues/175#issuecomment-2271327388

> Right now there are no sponsors, so the project is on hold. There is no timeline for releasing the 1.0.0 of AngleSharp.Css.
